app/models/helpers/helpers.rb

```diff
@@ -0,0 +1,13 @@
+module Helpers
```

Comment:
I'm all in favour of single-method modules (or classes). Mixing in a `Helpers` module with a dozen different unrelated methods into a model just because it needs e.g. date validation creates unnecessary bloat. It can also create difficult-to-track problems if the model tries to implement a method with the same name as a method in a module. Including a small single-purpose `DateValidator` module, on the other hand, minimizes such conflicts and makes it much clearer what method(s) a class inherits from the module.